src/components/global/Column.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
```

Comment:
This shouldn't really be a global component, as it can only be used in the task board view. It should also be renamed to be more specific, e.g. TaskBoardColumn.
Any component in the global subfolder is intended to be used across multiple different pages.